[2025-04-05T10:45:12.602+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-05T10:45:12.642+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: elt_pipeline.get_coin_data manual__2025-04-05T10:45:11.558526+00:00 [queued]>
[2025-04-05T10:45:12.653+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: elt_pipeline.get_coin_data manual__2025-04-05T10:45:11.558526+00:00 [queued]>
[2025-04-05T10:45:12.654+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-04-05T10:45:12.672+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): get_coin_data> on 2025-04-05 10:45:11.558526+00:00
[2025-04-05T10:45:12.683+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=236) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-05T10:45:12.682+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'elt_pipeline', 'get_coin_data', 'manual__2025-04-05T10:45:11.558526+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/elt_pipeline.py', '--cfg-path', '/tmp/tmpasuemu9e']
[2025-04-05T10:45:12.684+0000] {standard_task_runner.py:63} INFO - Started process 261 to run task
[2025-04-05T10:45:12.684+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask get_coin_data
[2025-04-05T10:45:12.708+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-05T10:45:12.761+0000] {task_command.py:426} INFO - Running <TaskInstance: elt_pipeline.get_coin_data manual__2025-04-05T10:45:11.558526+00:00 [running]> on host d56545bf874e
[2025-04-05T10:45:12.874+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='elt_pipeline' AIRFLOW_CTX_TASK_ID='get_coin_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-05T10:45:11.558526+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-05T10:45:11.558526+00:00'
[2025-04-05T10:45:12.875+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-05T10:45:13.072+0000] {logging_mixin.py:188} INFO - Error fetching data: 429 Client Error: Too Many Requests for url: https://api.coincap.io/v2/assets/bitcoin/history?interval=h1&start=1609459200000&end=1609545599999
[2025-04-05T10:45:13.073+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-04-05T10:45:13.076+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-05T10:45:13.090+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=elt_pipeline, task_id=get_coin_data, run_id=manual__2025-04-05T10:45:11.558526+00:00, execution_date=20250405T104511, start_date=20250405T104512, end_date=20250405T104513
[2025-04-05T10:45:13.119+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-05T10:45:13.148+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-05T10:45:13.153+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
